build/Dockerfile-go1.22.1-solc0.8.13-ubuntu-20.04

```diff
@@ -12,13 +12,13 @@ RUN cd /solidity && touch prerelease.txt
 RUN cd /solidity && make solc
 RUN cd /solidity && install -s solc/solc /usr/bin
 
-FROM ubuntu:20.04 as go_builder
+FROM ubuntu:22.04 as go_builder
 RUN apt update && apt install -y software-properties-common
 RUN add-apt-repository ppa:longsleep/golang-backports -y && \
     apt install -yq tzdata && \
     ln -fs /usr/share/zoneinfo/America/New_York /etc/localtime && \
     dpkg-reconfigure -f noninteractive tzdata
-RUN apt install -y build-essential bash gcc musl-dev openssl wget golang golang-1.17
+RUN apt install -y build-essential bash gcc musl-dev openssl wget golang golang-1.23
```

Comment:
1. Kaia does not build with go1.23 at the moment.
2. Which go version is actually used between `apt install golang-1.23` and `wget go1.22.1.src.tar.gz`?